Makefile

```diff
@@ -0,0 +1,45 @@
+SYMFONY_ENV = dev
+PHP_RUN = php7.4
+COMPOSER_PATH = /usr/local/bin/composer2
+ifeq ("$(wildcard $(COMPOSER_PATH))","")
+	COMPOSER_PATH = /usr/local/bin/composer
+endif
+COMPOSER_RUN = $(PHP_RUN) $(COMPOSER_PATH)/composer.phar
+
+.PHONY: help
+help: ## List of all available commands
+	@awk 'BEGIN {FS = ":.*?## "} /^[a-zA-Z_-]+:.*?## / {printf "\033[36m%-15s\033[0m %s\n", $$1, $$2}' $(MAKEFILE_LIST)
+
+.DEFAULT_GOAL := help
+
+.PHONY: php-version
+php-version: ## See PHP version needed for this project
+	@echo $(PHP_RUN)
+
+.PHONY: vendor
+vendor: ## Run composer install
+	$(COMPOSER_RUN) install
+
+.PHONY: assets-node
+assets-node:
+	sudo n auto
```

Comment:
I do not think we should need to use `sudo` here?

Reply:
Yes, it shouldn't be needed if `n` was installed correctly.

Reply:
If we use `nvm`, then `n` should not be needed, can't we just use `node` binary?

In any case, maybe we should have both commands: `assets-node` for use with `node` binary, and `assets-n` for use with `n` ?